src/forms/create_ticket.ts

```diff
@@ -154,7 +160,7 @@ class FormFields extends BaseFormFields {
             case ZDFieldTypes.Muliselect: {
                 f.type = AppFieldTypes.STATIC_SELECT;
                 const options = this.isSystemField(field) ? field.system_field_options : field.custom_field_options;
-                f.options = makeOptions(options);
+                f.options = makeOptions(options as ZDFieldOption[]);
```

Comment:
Why does `makeOptions` not return an `ZDFieldOption[]`?

Reply:
Essentially, this function is converting a `Zendesk` Form Field to an `Apps` `AppField` 

`makeOptions` accepts `ZDFieldOption[]` and returns options for an `AppField`.  Does that make sense?

Reply:
I've misread the code.

Still, I think there is some case handling missing there. If `options` is `undefined` that line with throw an error because `field.system_field_options` might be `undefined`. Can that case happen?

Reply:
You cannot save a drop-down field in Zendesk without at least one option.  I'm not sure the best way to handle the check for that case.  If options are undefined, I suppose that I could just not show that field?

Reply:
Ok. I added a check for defined options and if they don't exist, the field is added, but without any options. Again, this case should never occur because of the zendesk restrictions when created a drop box.